File: 1672-richest-customer-wealth/1672-richest-customer-wealth.go
func maximumWealth(accounts [][]int) int {
    var answer int
    for _, v := range accounts {
        wealth := wealth(v)
        if wealth > answer {
            answer = wealth
        }
    }
    return answer
}

func wealth(account []int) int {
    var answer int
    for _, v := range account {
        answer += v
    }
    return answer
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int answer = 0;
        for (int[] account: accounts) {
            int wealth = Solution.wealth(account);
            if (wealth > answer) {
                answer = wealth;
            }
        }
        return answer;
    }

    static public int wealth(int[] account) {
        int wealth = 0;
        for (int money: account) {
            wealth += money;
        }
        return wealth;
    }
}
